101-automation-configuration/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+﻿{
```

Comment:
remove the params that have valid defaultValues

Reply:
fixed, thank you